library/src/main/java/com/heinrichreimersoftware/materialintro/app/SlideFragment.java

```diff
@@ -58,20 +58,30 @@ public void removeOnNavigationBlockedListener(OnNavigationBlockedListener listen
     }
 
     @Override
-    public boolean goToSlide(int position) {
-        return getIntroActivity().goToSlide(position);
+    public boolean goToSlide(int position, boolean forceScroll) {
+        return getIntroActivity().goToSlide(position, forceScroll);
     }
 
     @Override
     public boolean nextSlide() {
         return getIntroActivity().nextSlide();
     }
 
+    @Override
+    public boolean forceNextSlide() {
+        return getIntroActivity().forceNextSlide();
+    }
+
     @Override
     public boolean previousSlide() {
         return getIntroActivity().previousSlide();
     }
 
+    @Override
+    public boolean forcePreviousSlide() {
```

Comment:
I would remove the duplicated code in favor of a `previousSlide(boolean forceScroll)` method.
The old `previousSlide()` method should then call `previousSlide(false);`